packages/utilities/src/utilities/stripHTML/stripHTML.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Utility returns the text stripping all html tags from it.
+ *
+ * @example
+ * import { stripHTML } from '@carbon/ibmdotcom-utilities';
+ *
+ * content = stripHTML(this.innerHtml);
+ *
+ * @param {string} content with html tags
+ * @returns {null} content without html tags
+ */
+const stripHTML = content => content.replace(/(<([^>]+)>)/gi, '');
```

Comment:
@RaphaelAmadeu can you create some light unit tests around this utility?

Reply:
@jeffchew i've added this in the latest commit